Framework/API/inc/MantidAPI/Workspace.h

```diff
@@ -117,7 +117,7 @@ class MANTID_API_DLL Workspace : public Kernel::DataItem {
   /// The history of the workspace, algorithm and environment
   std::unique_ptr<WorkspaceHistory> m_history;
   /// Storage mode of the Workspace (used for MPI runs)
-  Parallel::StorageMode m_storageMode{Parallel::StorageMode::Cloned};
+  Parallel::StorageMode m_storageMode{Parallel::StorageMode::MasterOnly};
```

Comment:
Maybe, though we have a wide variety of workspaces, with various other constructor arguments, and some are created by factories.